### PR TITLE
Json2yaml

### LINKED ISSUE #17: Convert JSON to YAML
# Description

Create a script that takes a JSON file as input and converts it into a YAML file.

## Type of issue

- [x] Feature (New Script)

## Checklist:

- [x] I have read the project guidelines.
- [x] I have checked all the existing projects, before submitting a new project issue.
- [x] I have checked previous issues to avoid duplicates.
- [x] This issue will be meaningful for the project.

### PR DESCRIPTION
**Pull request by mendoza**
# Description

It converts a json file to a yaml file

Fixes #17

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [X] I have named my files and folder, according to this project's guidelines.
- [X] My code follows the style guidelines of this project.
- [X] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [X] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [X] I have included a requirements.txt file (if external libraries are required.)
- [X] My changes do not produce any warnings.
- [X] I have added a working sample/screenshot of the script.

### CODE REVIEW

File: Scripts/Miscellaneous/json2yaml/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Python script that provides the download and upload speed of your Wifi
+A simple Python script that uses 'pyyaml' module to conver an input json file into a output yaml file.
```
Comment (tusharnankani) [inline Scripts/Miscellaneous/json2yaml/README.md:2]: Leave a line after every heading

File: Scripts/Miscellaneous/json2yaml/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+### Prerequisites
+You will need to install pyyaml. 
+The installation commands are provided in requirements.txt
+
+### How to run the script
+First you have to go to the json2yaml directory. Run the following command once you are in project directory
+
+```cd Scripts/Miscellaneous/json2yaml```
+
```
Comment (tusharnankani) [inline Scripts/Miscellaneous/json2yaml/README.md:12]: use **\```bash** for bash commands and write the commands in the next line, and close **```** it, in the line after that.
```
```bash
<commands>
\```
```

**Review by tusharnankani — CHANGES_REQUESTED** [review]

**Review by tusharnankani — COMMENTED** [review]
Please fix the Readme errors.
And, comment your code for better understanding.

The script works fine. 👍🏻 
Good work.

**Review by tusharnankani — APPROVED** [review]

**Review by chavarera — CHANGES_REQUESTED** [review]
code is working fine

Suggestions
Add Helper functions instead if writing statement by statement
`A helper function is a function that performs part of the computation of another function. Helper functions are used to make your programs easier to read by giving descriptive names to computations. They also let you reuse computations, just as with functions in general.`

**Comment by mendoza** [comment]
Is something missing?

**Review by AdityaJ7 — APPROVED** [review]

**Pull request merged**